features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/Comments/Comment.jsx

```diff
@@ -367,6 +509,33 @@ class Comment extends React.Component {
                                                     </Box>
                                                 </>
                                             ))}
+                                            {comment.replies && comment.replies.count < comment.replies.pagination.total
+                                                && (
+                                                    <div className={classes.contentWrapper}>
+                                                        <Grid container spacing={4} className={classes.root}>
+                                                            <Grid item>
+                                                                <Typography className={classes.verticalSpace} variant='body1'>
+                                                                    <a
+                                                                        className={classes.link + ' ' + classes.loadMoreLink}
+                                                                        onClick={() => this.handleLoadMoreReplies(comment)}
```

Comment:
why do we need to pass the whole comment? I think we only need the parent/root comment ID to get more comments?

Reply:
We pass the entire comment cause we use it to update the corresponding entry of the retrieved comment list[1]. The method used should be changed to `updateComment` instead of `commentsUpdate`. Will fix it.

[1]. https://github.com/wso2/carbon-apimgt/pull/10226/files#diff-378d58c4ccae87a0211f8c8a5ef12edcb62e46f9e81d602dd2a53709bc852451R291

Reply:
changed update logic with https://github.com/wso2/carbon-apimgt/pull/10226/commits/aeb23fd99c3c675ca4ba8390bf2933f471144833